### PR TITLE
Feature/2021 label templates

### PR DESCRIPTION
**Pull request by matteason**

### CODE REVIEW

File: src/main/resources/labels/commercial-refrigeration-direct-sales/commercial-refrigeration-direct-sales.svg
Comment (jamesbarnett91) [inline src/main/resources/labels/commercial-refrigeration-direct-sales/commercial-refrigeration-direct-sales.svg:175]: It looks like we've lost the multi line supplier name and model name in the new labels. Can they still take the same approximate number of characters? They do seem a bit wider.
If not it may be tricky to change the length validation on categories with both old multi-line labels and new single line labels active as they would currently share the same generic supplier and name fields.

**Review by jamesbarnett91 — APPROVED** [review]

File: src/main/resources/labels/commercial-refrigeration-direct-sales/commercial-refrigeration-direct-sales.svg
Comment (matteason) [reply src/main/resources/labels/commercial-refrigeration-direct-sales/commercial-refrigeration-direct-sales.svg:175]: Yeah good point. We might be able to go up to two rows for some of them - there's nothing in the regulations that prevent that. Though for electronic displays the QR code is taller than the flag which means there's not room for the model number to go over two lines. Probably going to be easier to test and adjust once they're in the generator so I'll merge and look at these when we're doing the form changes

**Pull request merged**
